orderly_web/start.py

```diff
@@ -113,9 +113,10 @@ def web_container_config(cfg, container):
     opts = {"app.port": str(cfg.web_port),
             "app.name": cfg.web_name,
             "app.email": cfg.web_email,
-            "app.github_org": cfg.web_auth_github_org,
-            "app.github_team": cfg.web_auth_github_team,
-            "app.auth": str(cfg.web_auth_fine_grained).lower(),
+            "auth.github_org": cfg.web_auth_github_org,
```

Comment:
Opportunistic fix so I could open the app and see it working. Discovered a bug with static files ( https://vimc.myjetbrains.com/youtrack/issue/mrc-261), but was able to confirm adding permissions works.